third-party/tclap/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+cmake_minimum_required( VERSION 3.10 )
```

Comment:
Why do we limit the minimum version?

Reply:
CMake makes you put a minimum-required in the beginning. Maybe not for everything, but I recall it complaining.
Anyway, U18 is our minimum, so 3.10.

What's most interesting to me is that the static_analysis hasn't complained...!
And when I check it, it says:
`The test PASSED, current CMake version is VERSION 3.10.0 and it is not higher than VERSION 3.8.0`

LOL?

Reply:
I removed it, let's see what happens.

Reply:
Looks OK now